patches/ui-views-accessibility-accessibility_paint_checks.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ui/views/accessibility/accessibility_paint_checks.cc b/ui/views/accessibility/accessibility_paint_checks.cc
+index fbda0e2df78ce64d2cd8eb4467228493235837c6..fd09ff50aa7b4911068bddec70848c6e4fe295ab 100644
+--- a/ui/views/accessibility/accessibility_paint_checks.cc
++++ b/ui/views/accessibility/accessibility_paint_checks.cc
+@@ -33,6 +33,7 @@ void RunAccessibilityPaintChecks(View* view) {
+   // some additional fields are processed and set there.
+   ui::AXNodeData node_data;
+   view->GetViewAccessibility().GetAccessibleNodeData(&node_data);
++  node_data.SetNameExplicitlyEmpty();
```

Comment:
https://github.com/brave/brave-browser/issues/748
> Remove the patch once upstream fixes the bug

I wonder if we should remove this patch at some point?
I assume it checks for accessibility bugs which is a good thing if we want our browser to be accessible.

Reply:
I've assigned https://github.com/brave/brave-browser/issues/748 to @cdesouza-chromium and we'll investigate this while working on cr107+, since it's not strictly tied to a Chromium bump and we're a little bit pressed on time if that's OK with you.